addons/gbase/templates/cmpd-coordinator.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: ComponentDefinition
+metadata:
+  name: {{ include "gbase.componentDefName" . }}-coord
+  labels:
+    {{- include "gbase.labels" . | nindent 4 }}
+spec:
+  provider: kubeblocks.io
+  description: gabse 8c coordinator node
+  serviceKind: gbase
+  serviceVersion: 5.0.0
+  configs:
+    - name: gbase-config
+      templateRef: {{ include "gbase.cmConfigName" . }}
+      namespace: {{ .Release.Namespace }}
+      volumeName: gbase-config
+      defaultMode: 0444
+  scripts:
+    - name: gbase-scripts
+      templateRef: {{ include "gbase.cmScriptsName" . }}
+      namespace: {{ .Release.Namespace }}
+      volumeName: scripts
+      defaultMode: 0555 
+  updateStrategy: Parallel
+  podManagementPolicy: Parallel
+  runtime:
+    nodeSelector:
+      gbase: "true"
+    containers:
+      - name: gbase
+        imagePullPolicy: {{ default "IfNotPresent" .Values.image.pullPolicy }}
+        command: ["/usr/sbin/init"]
+        lifecycle:
+          postStart:
+            exec:
+              command: ["/scripts/node_setup.sh"]
+        securityContext:
+          privileged: true
+          runAsUser: 0
+        volumeMounts:
+          - mountPath: /config
+            name: gbase-config
+          - mountPath: /data
+            name: data
+          - name: scripts
+            mountPath: /scripts
+          - name: ssh-key
+            mountPath: /ssh-key
+            readOnly: true
+    volumes:
+    - name: ssh-key
+      secret:
+        secretName: gbase-ssh-rsa-secret
```

Comment:
gbase-ssh-rsa-secret, the secret name is a constant.
we can define `volumeMounts` in cmpd, and define volumes in 'cluster.spec.componentSpecs.volumes'.  and the secret name could be any name given by user.